tensorflow/python/kernel_tests/signal/fft_ops_test.py

```diff
@@ -158,6 +158,31 @@ def _tfIFFTForRank(self, rank):
     else:
       raise ValueError("invalid rank")
 
+  #test cases for fftshift and ifftshift
+  def testDefinition(self):
+    x = [0, 1, 2, 3, 4, -4, -3, -2, -1]
+    y = [-4, -3, -2, -1, 0, 1, 2, 3, 4]
+    self.assertTrue((fft_ops.fftshift(x).numpy() ==  y).all())
+    self.assertTrue((fft_ops.ifftshift(y).numpy() ==  x).all())
+    x = [0, 1, 2, 3, 4, -5, -4, -3, -2, -1]
+    y = [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4]
+    self.assertTrue((fft_ops.fftshift(x).numpy() == y).all())
+    self.assertTrue((fft_ops.ifftshift(y).numpy() ==  x).all())
```

Comment:
could you also include a test verifying that numpy.fft.fftshift/ifftshift produces the same results?

Reply:
Added as a separate test case in the test class [testNumpyCompanibility](https://github.com/tensorflow/tensorflow/blob/03be2ce9b4f35e54135163993af62cb45e82e15d/tensorflow/python/kernel_tests/signal/fft_ops_test.py#L583-L597). Sir, required Any name change or merging this function with definition?

Reply:
sir, how can i add these functions in the official docs [tf.signals](https://www.tensorflow.org/api_docs/python/tf/signal)